### PR TITLE
Avoid use of prefix in Swift functions

### PR DESCRIPTION
**Pull request by alexissan**
Added new rule and some example to avoid using functions prefixes in Swift code

### CODE REVIEW

**Comment by AntonPalich** [comment]
@alexissan What if I expose a function to Objective-C using `@objc` and call it from Objective-C. Should I use prefix or not in this case?

File: README.markdown [hunk 1]
```diff
@@ -108,6 +108,35 @@ import SomeModule
...
+}
+```
+
+**Preferred:**
+```swift
+extension MyExtension where Base: UIColor {
+    func colorToImage() -> UIImage {
+        let color = self.base
+
+        // function to convert image to color
```
Comment (AntonPalich) [inline README.markdown]: "color to image"

Comment (alexissan) [reply README.markdown]: will change

**Comment by alexissan** [comment]
@AntonPalich in this case you should do it, but in this case you are writing Objective-C code inside of a Swift "macro", should I add this case as an exception?

**Comment by AntonPalich** [comment]
@alexissan I think yes, otherwise it's not clear that there is exist an exception from the rule

**Comment by dive** [comment]
@AntonPalich, @alexissan, I think for Objective C we should use https://github.com/badoo/objective-c-style-guide conventions. This rule has nothing to do with Objective C.

**Comment by alexissan** [comment]
@dive do you suggest to remove this exception to the rule?
cc @AntonPalich

**Comment by AntonPalich** [comment]
@dive, @alexissan  This exception is related to Swift code. I see no reasons to remove it. We write `@objc` in Swift code, not in Objective-C

**Pull request merged**